src/localisation/localisation.h

```diff
@@ -88,4 +88,7 @@ static inline void set_format_arg_body(uint8 *args, size_t offset, uintptr_t val
 #define set_map_tooltip_format_arg(offset, type, value) \
 	set_format_arg_body(gMapTooltipFormatArgs, offset, (uintptr_t)value, sizeof(type))
 
+money32 string_to_money(char * string_to_monetise);
```

Comment:
Why are these functions declared at the bottom of the file, and not together with the other function declarations?